Tests/HAPTests/TLV8Tests.swift

```diff
@@ -14,12 +14,19 @@ class TLV8Tests: XCTestCase {
 
     func test() {
         let publicKey = Data(repeating: 0, count: 256) + Data(repeating: 1, count: 256) + Data(repeating: 2, count: 256)
-        let original: PairTagTLV8 = [
-            .publicKey: publicKey
+        let original: [PairTagTLV8] = [
+            (.publicKey, publicKey)
         ]
         let encoded = encode(original)
-        let decoded: PairTagTLV8 = try! decode(encoded)
-        XCTAssertEqual(original, decoded)
```

Comment:
Can we make the 'new' PairTagTLV8 equatable, what would be needed for that?

Reply:
Unfortunately Swift 4 does not permit tuples to implement `Equatable` (or any other Protocol), but it is possible to add an `==`  operator to the PairTagTVL8 array, which I’ve now implemented in TVL8.swift. Even though it has an equality operator, it still can’t conform to the `Equatable` protocol, but it does simplify the test case code to         
`XCTAssertTrue(original == decoded)`